test/models/project_test.rb

```diff
@@ -0,0 +1,44 @@
+require "test_helper"
+
+class ProjectTest < ActiveSupport::TestCase
+  test "valid with user" do
+    user = User.create(name: "Ali", mobile: "0900", email: "a2@g.com", password_digest: "123", address: "ajhsncjfacnaje")
+    assert user.valid?
+
+    assert_empty user.errors
+  end
+
+  test "invalid without user" do
+    user = User.create()
+    assert user.valid?
```

Comment:
same as above, you are testing user not project.

Reply:
done